### ISSUE TITLE
Remove collaborator

### ISSUE THREAD

**Issue by VillainsRule4000** [Hankypoo7/The-blooket-hacks#145]
@v8f is being toxic. Please remove him as a collaborator. He is violating our Code of Conduct.

I have removed all instances, but I have screenshots if needed.

**Comment by VillainsRule4000** [comment]
@Hankypoo7 This is critical...

**Comment by Hankypoo7** [comment]
ok

**Comment by v8f** [comment]
ok then, barely was but ok

**Comment by Hankypoo7** [comment]
i removed him what did he do

**Comment by v8f** [comment]
all i did was just state my opinion on guilded

**Comment by v8f** [comment]
and said like one thing

**Comment by Hankypoo7** [comment]
what did you say

**Comment by v8f** [comment]
hell

;lmao

**Comment by v8f** [comment]
apparently thats a bad word
<br>
doubt

**Comment by Hankypoo7** [comment]
um

**Comment by Hankypoo7** [comment]
@VillainsRule4000 is that all?

**Comment by v8f** [comment]
curious on why he wanted to use guilded
ive only been barely toxic like a few things and thats it

?

**Comment by v8f** [comment]
also no im not violating no code of conduct im just trying to state my opinion on stuff and do stuff

:shrug:

**Comment by v8f** [comment]
theres only a few things on coc.md
* Public or private harassment
* Publishing others' private information, such as a physical or email
  address, without their explicit permission
* Any racist comments or any comments to make people feel bad about themselves
* Other conduct which could reasonably be considered inappropriate in a
  professional setting
  none of that i have done ?

**Comment by v8f** [comment]
is stating my opinion on guilded considered instnatly breakiing everything you guys have and saying hell breaks everything in the world? no
deal with it
i can say a few words

**Comment by VillainsRule4000** [comment]
@v8f "blame your network or whatever else."

**Comment by Hankypoo7** [comment]
yeah villancoder if thats all he said thats not really bad or anything

**Comment by Hankypoo7** [comment]
whats going on

**Comment by Hankypoo7** [comment]
so what should i do,
i took away v8fs perms for now

**Comment by Hankypoo7** [comment]
ok
